application-admintools-default/pom.xml

```diff
@@ -30,13 +30,21 @@
   <packaging>jar</packaging>
   <name>Admin Tools Application (Pro) - Default</name>
   <description>Admin Tools Application (Pro) - Default</description>
```

Comment:
Since you have also the unit tests added, you need to add a property in the pom with the code coverage. For example:
```
<properties>
    <xwiki.jacoco.instructionRatio>0.85</xwiki.jacoco.instructionRatio>
</properties>
``` 
This will mention how much of your code is tested. The nice thing about this property is that you cannot go below the value defined here. So, whenever java code is added, you are forced to add also unit tests, so to not go below the defined value, otherwise the build will fail :)